.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -6,6 +6,7 @@ Briefly describe the new feature or fix here
 
 Was a software definition added? Or a new version to an existing definition?
 - [ ] (Chef employee) Add software tarball/gz/zip/whatevs to the opscode-omnibus-cache S3 bucket in preprod
+- [ ] (Chef employee) If this is not a minor change, verify with an ad-hoc build of Automate, Chef-DK, or Chef Server (if applicable -- ask @londo to find out).
```

Comment:
@scotthain it is a bot that @stevendanna develop. Try `@londo package has-dep postgresql`